library/core/src/intrinsics/mir.rs

```diff
@@ -270,12 +278,15 @@ macro_rules! define {
         pub fn $($sig)* { panic!() }
     }
 }
-
 define!("mir_return", fn Return() -> BasicBlock);
 define!("mir_goto", fn Goto(destination: BasicBlock) -> BasicBlock);
 define!("mir_unreachable", fn Unreachable() -> BasicBlock);
-define!("mir_drop", fn Drop<T>(place: T, goto: BasicBlock));
-define!("mir_call", fn Call(call: (), goto: BasicBlock));
+define!("mir_continue", fn Continue());
+define!("mir_resume", fn Resume());
+define!("mir_terminate", fn Terminate(reason: UnwindTerminateReason));
```

Comment:
Here you are mixing `UnwindAction` and `Termintor`, that is quite confusing.

Reply:
Yeah, I renamed the `TerminatorKind` recently to include `Unwind` in the name; would be good to do the same in custom MIR.

Reply:
But what I meant above is that you have a `define!` list of terminator, and then mix in an `UnwindAction` in the middle. The list should probably be grouped by type with a comment indicating which type the next group has.